zipline/__main__.py

```diff
@@ -227,9 +235,18 @@ def run(ctx,
         broker,
         broker_uri,
         state_file,
+        broker_list,
         realtime_bar_target):
     """Run a backtest for the given algorithm.
     """
+
+    if broker_list:
+        click.echo("Supported brokers:")
+        for module_loader, name, ispkg in pkgutil.iter_modules(brokers.__path__):
+            if name != 'broker':
+                print name
```

Comment:
This line breaks python 3 support (see travis log with python 3 runs).
I'd suggest using `click.echo()`, which is consistent with the rest of this file